docs/tutorials/docker/getting-started.md

```diff
@@ -1,198 +1,297 @@
 
-## Creating a Smart Contract
+![](./images/docker-macos-hello-world.png)
 
-Create a directory in the `ultra_workdir` directory called `contracts` with a file inside called `hello.cpp`.
-- You can do it either on your host machine (Windows/Linux) or inside the docker image using your editor of choice (`nano` is preinstalled, other editors require manual installation)
-- You also have an option to use [VSCode Environment](./development-environment.md)
+You likely want to clean up the created container, you will have no use for it and it will just clog the container list. To delete it, simply click on the garbage can icon next to the `hello-world` container:
 
-```cpp
-#include <eosio/eosio.hpp>
+![](./images/docker-windows-delete-hello-world.png)
 
-using namespace eosio;
+::: info
+After installing and opening the docker application, you should be able to follow the same usage instructions as for Linux.
+:::
+
+::::
+
+## Obtaining the Docker image
 
-class [[eosio::contract]] hello : public eosio::contract {
-  public:
-      using contract::contract;
+After you have installed Docker on your machine, you should be able to pull the Docker image down from quay.io:
 
-      [[eosio::action]]
-      void world( eosio::name name ) {
-        print("Hi, ", name);
-      }
-};
+```sh
+docker pull quay.io/ultra.io/3rdparty-devtools:latest
 ```
 
-### Compiling a Smart Contract
+You will also want to use this command to update the Docker image you have locally.
+
+The Docker image we provide is located at [our official quay.io repository](https://quay.io/ultra.io/3rdparty-devtools). If you are interested, you can check there to see if there is a new version of the image available, or use an older version if needed.
+
+## Running the Developer Tools Docker image
 
-Inside of the docker image (using the terminal that is attached to the `ultra` container) navigate into the `contracts` directory, and run the following command.
+To start the Docker container with the cloned Developer Tools Docker image use _one_ of the following commands appropriate for your operating system:
 
+::: details Windows
+```sh
+docker run -dit --name ultra -p 8888:8888 -p 9876:9876 -v %cd%/ultra_workdir:/opt/ultra_workdir --name ultra quay.io/ultra.io/3rdparty-devtools:latest
 ```
-mkdir -p /opt/ultra_workdir/contracts && cd /opt/ultra_workdir/contracts
+
+You will be asked if you are okay with sharing the working directory with the Docker container. Only the content of the directory specified will be shared, and not any other content of your drive.
+
+![](./images/docker-windows-allow-sharing.png)
+:::
+
+::: details Linux
+```sh
+docker run -dit --name ultra -p 8888:8888 -p 9876:9876 -v ~/ultra_workdir:/opt/ultra_workdir --name ultra quay.io/ultra.io/3rdparty-devtools:latest
 ```
+:::
+
```

Comment:
Shouldn't we add macOS as well. I suspect we need to use the same command on macOS than in Linux, so
the fix could be something like this 
```sh
::: details Linux/macOS
```